pkg/chunk/chunk_store_utils.go

```diff
@@ -102,7 +102,7 @@ type decodeResponse struct {
 func NewChunkFetcher(cacher cache.Cache, cacheStubs bool, storage Client) (*Fetcher, error) {
 	c := &Fetcher{
 		storage:        storage,
-		cache:          cacher,
+		cache:          cache.NewSnappy(cacher),
```

Comment:
This likely doesn't require compression.

Reply:
@gouthamve What are the parameters to take into consideration when a cache requires compression or not?

Reply:
Well the idea is that the chunks are already compressed using our own custom encoding format and compression on top of that will give us little gains.

Reply:
IMO only the results cache would really benefit from this.